src/algebra/ordered_monoid.lean

```diff
@@ -494,9 +494,19 @@ by norm_cast
 lemma coe_bit1 [add_semigroup α] [has_one α] {a : α} : ((bit1 a : α) : with_bot α) = bit1 a :=
 by norm_cast
 
-@[simp] lemma bot_add [ordered_add_comm_monoid α] (a : with_bot α) : ⊥ + a = ⊥ := rfl
+@[simp] lemma bot_add [add_monoid α] (a : with_bot α) : ⊥ + a = ⊥ := rfl
 
-@[simp] lemma add_bot [ordered_add_comm_monoid α] (a : with_bot α) : a + ⊥ = ⊥ := by cases a; refl
+@[simp] lemma add_bot [add_monoid α] (a : with_bot α) : a + ⊥ = ⊥ := by cases a; refl
+
+@[simp] lemma add_eq_bot_iff [add_monoid α] {m n : with_bot α} :
+  m + n = ⊥ ↔ m = ⊥ ∨ n = ⊥ :=
```

Comment:
Can you make the name consistent with `add_eq_top`? 
Also, is the proof just `add_eq_top`?
Also, are the instances `add_semigroup` for these 3 lemmas sufficient?